sdk/python/feast/feature_server.py

```diff
@@ -117,16 +134,34 @@ def push(body=Depends(get_body)):
         try:
             request = PushFeaturesRequest(**json.loads(body))
             df = pd.DataFrame(request.df)
+            actions = []
             if request.to == "offline":
                 to = PushMode.OFFLINE
+                actions = [AuthzedAction.WRITE_OFFLINE]
             elif request.to == "online":
                 to = PushMode.ONLINE
+                actions = [AuthzedAction.WRITE_ONLINE]
             elif request.to == "online_and_offline":
                 to = PushMode.ONLINE_AND_OFFLINE
+                actions = WRITE
             else:
                 raise ValueError(
                     f"{request.to} is not a supported push format. Please specify one of these ['online', 'offline', 'online_and_offline']."
                 )
+
+            feature_views = store.list_feature_views(
+                allow_cache=request.allow_registry_cache
+            )
+            stream_feature_views = store.list_stream_feature_views(
+                allow_cache=request.allow_registry_cache
+            )
+
```

Comment:
You also need to apply this [filter](https://github.com/feast-dev/feast/blob/master/sdk/python/feast/feature_store.py#L1408C9-L1416C10) before you assert permissions.

Reply:
added filter please take another [look](https://github.com/RHEcosystemAppEng/feast/compare/a9cb218e7488f72187aacc7509de32ec5ab94b35..4668c616260198de335cbf312c95af715f507ef2#diff-47bf85967b55f577ebd4586f9a93f2cc0c9433270bd67b21ddb33e89fd464167R158-R175)